Src/Generic/Viewables/Button.swift

```diff
@@ -15,13 +15,16 @@ import UIKit
 enum ButtonType {
     case standard(backgroundColor: HedvigColor, textColor: HedvigColor)
     case pillTransparent(backgroundColor: HedvigColor, textColor: HedvigColor)
+    case iconTransparent(textColor: HedvigColor, icon: ImageAsset?)
```

Comment:
`ImageAsset` should probably not be an Optional right, because that wouldn't be a valid case?

Reply:
Correct!